manifests/repo/debian.pp

```diff
@@ -2,66 +2,30 @@
 #
 # === Parameters
 #
-# [*location*]
-#   Location of the apt repository
-#
-# [*release*]
-#   Release of the apt repository
-#
-# [*repos*]
-#   Apt repository names
-#
 # [*include_src*]
 #   Add source source repository
 #
 # [*key*]
 #   Public key in apt::key format
 #
-# [*dotdeb*]
-#   Enable special dotdeb handling
-#
-# [*sury*]
-#   Enable special sury handling
-#
 class php::repo::debian (
-  $location     = 'https://packages.dotdeb.org',
-  $release      = 'wheezy-php56',
-  $repos        = 'all',
   $include_src  = false,
   $key          = {
-    'id'     => '6572BBEF1B5FF28B28B706837E3F070089DF5277',
-    'source' => 'http://www.dotdeb.org/dotdeb.gpg',
+    'id'     => '15058500A0235D97F5D10063B188E2B695BD4743',
+    'source' => 'https://packages.sury.org/php/apt.gpg',
   },
-  $dotdeb       = true,
-  $sury         = true,
 ) {
   assert_private()
 
   include 'apt'
 
-  apt::source { "source_php_${release}":
-    location => $location,
-    release  => $release,
-    repos    => $repos,
+  apt::source { 'source_php_sury':
+    location => 'https://packages.sury.org/php/',
```

Comment:
quite the same as the comment above, but this should be customizable or only drop-in if the requested php version cannot be fullfilled by the debian repo

Reply:
fair point i ll rework

Reply:
done